pkg/volume/glusterfs/glusterfs.go

```diff
@@ -656,6 +659,12 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 			cfg.user = v
 		case "restuserkey":
 			cfg.userKey = v
+		case "gid":
+			reqGid, err = strconv.Atoi(v)
```

Comment:
@liggitt thats a good thought !  yeah, I do think either mount/unmount or attach/detach interface may need it as FSGroup/SupplementalGroupID form, so better to have it as an actual field than handling through annotations.